### PR TITLE
Preact/React developer tools documentation.

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update

## Description

Some documentation updates for debugging Preact components after discussing it on Slack today.

## Added to documentation?

- [x] docs.dev.to
- [ ] readme
- [ ] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

Winston from New Girl saying he can open a document

### CODE REVIEW

File: docs/frontend/debugging.md [hunk 1]
```diff
@@ -9,22 +9,20 @@ title: Debugging
 Browsers ship with their own developer tools. These are amazing tools to help
 you debug your web application. Consider learning how to use them.
 
-- [Chrome Developer
-  Tools](https://developers.google.com/web/tools/chrome-devtools)
+- [Chrome Developer Tools](https://developers.google.com/web/tools/chrome-devtools)
```
Comment (nickytonline) [inline docs/frontend/debugging.md]: Prettier in action.

**Comment by nickytonline** [comment]
I think it’ll be a while before the testing tools support Preact X (at least a month), and @vaidehi was asking about this yesterday. I also imagine it might be confusing for the community which is why I wanted to give a little explanation.

**Comment by rhymes** [comment]
@nickytonline makes sense! what if we just explain without everything after this sentence?

> The browser extension is available for Chrome and Firefox:

anyhow, I'm okay either way

**Comment by nickytonline** [comment]
> @nickytonline makes sense! what if we just explain without everything after this sentence?

Sounds good. I’ll make the change.

**Pull request merged**